### PR TITLE
fix: improve load policy line

### LINKED ISSUE #238: Trim inputs when adding policy/role
I'm using Casbin with Postgres.
I encountered a situation where I use addPolicy/addRoleForUser with parameters that have a whitespace prefix.
The problem is that when the policy is read, you do trim the values so upon next insertion of the same values (duplicate), Casbin
doesn't identify that it's duplicate, resulting in duplicate lines in the DB.

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

fixes #238

### CODE REVIEW

File: src/persist/helper.ts [hunk 1]
```diff
@@ -1,13 +1,23 @@
 import { Model } from '../model';
+import * as parse from 'csv-parse/lib/sync';
 
 export class Helper {
   public static loadPolicyLine(line: string, model: Model): void {
-    if (!line || line.trim() === '' || line.charAt(0) === '#') {
+    if (!line || line.charAt(0) === '#') {
```
Comment (Zxilly) [inline src/persist/helper.ts:6]: Maybe exist something like `[blank space]# This is a comment`

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: src/persist/helper.ts (hunk 1 above)
Comment (nodece) [reply src/persist/helper.ts:6]: good point.

**Review by Zxilly — APPROVED** [review]
LGTM, but plz format code with prettier

**Comment by nodece** [comment]
@Zxilly Thanks for your review.

**Comment by hsluoyz** [comment]
@EladProject please review.

**Review by EladProject — APPROVED** [review]

**Pull request merged**